test/test_torch.py

```diff
@@ -3389,13 +3411,21 @@ def test_clone_all_dtypes_and_devices(self):
         for device in torch.testing.get_all_device_types():
             for dt in torch.testing.get_all_dtypes():
                 x = torch.tensor((1, 1), dtype=dt, device=device)
+                if (device == 'cuda' and dt == torch.bfloat16):
+                    self.assertRaises(RuntimeError, lambda: x.clone())
```

Comment:
nit: for assertRaises, is there a message you can you set for when the tests fail?  It would be helpful to put something like "placeholder for future functionality -- check if you should update the test" so it's clear we don't actually intend for this to be the perfect behavior.